### PR TITLE
Collective Overview: include balance time series & make preview feature public

### PR DESCRIPTION
**Pull request by gustavlrsn**
For https://github.com/opencollective/opencollective/issues/7180
Requires https://github.com/opencollective/opencollective-api/pull/9789

# Description
- Uses the balance time series as the main chart that is displayed in the Collective Overview
- Fix for picking period dates to not use end date that is later than the current day
- Adds a feedback button and dismissable welcome popover message.
- Making the preview feature publicly available

# Screenshots

image

### CODE REVIEW

**Comment by gustavlrsn** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #10046** <a href="https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/10046?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/opencollective/opencollective-frontend/10046?utm_source=stack-comment).

File: components/dashboard/sections/overview/Metric.tsx [hunk 1]
```diff
@@ -42,9 +42,10 @@ export interface MetricProps extends React.ComponentPropsWithoutRef<'div'> {
   };
   helpLabel?: React.ReactNode;
   showCurrencyCode?: boolean;
+  showTimeSeries?: boolean;
   expanded?: boolean;
-  withTimeseries?: boolean;
   currency?: Currency;
+  isSnapshot?: boolean;
 }
 export function Metric({
```
Comment (kewitz) [inline components/dashboard/sections/overview/Metric.tsx:50]: From the creator of the tilting Virtual Card 🤐, I think we should replace the hover:scale animation with a simpler border color update.

**Review by kewitz — APPROVED** [review]
Looking good. I've tested with multiple collectives and couldn't spot a bug. The date axis and intervals seems to be working properly as well.

File: components/dashboard/sections/overview/Metric.tsx (hunk 1 above)
Comment (gustavlrsn) [reply components/dashboard/sections/overview/Metric.tsx:59]: Makes sense, the scaling is a bit jarring. I removed the scaling and added a ring/border on `focus-visible`, and keeping the shadow for hover.

**Pull request merged**